### PR TITLE
Add Todd and rename gsp to lucuma

### PR DESCRIPTION
**Pull request by cquiroz**
Let's make the plug-in names consistent with the other projects

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -1,18 +1,18 @@
 
 inThisBuild(
-  (homepage := Some(url("https://github.com/gemini-hlsw/sbt-gsp"))) +: gspPublishSettings
+  (homepage := Some(url("https://github.com/gemini-hlsw/sbt-lucuam"))) +: lucumaPublishSettings
```
Comment (rpiaggio) [inline build.sbt:3]: Typo

**Review by rpiaggio — APPROVED** [review]
LGTM

File: build.sbt [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
 )
 
 addSbtPlugin("de.heikoseeberger"         % "sbt-header"   % "5.6.0")
 addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.13")
 addSbtPlugin("ch.epfl.scala"             % "sbt-scalafix" % "0.9.20")
 
-lazy val sbtGsp = (project in file("."))
-  .disablePlugins(GspPlugin)
+lazy val lucumaGsp = (project in file("."))
```
Comment (swalker2m) [inline build.sbt:10]: Maybe it doesn't matter but `sbtLucuma` here I think?

File: build.sbt (hunk 1 above)
Comment (cquiroz) [reply build.sbt]: Thanks, it is now fixed

File: build.sbt (hunk 2 above)
Comment (cquiroz) [reply build.sbt]: Fixed

**Pull request merged**